cpp/tests/copying/slice_tests.cu

```diff
@@ -78,6 +78,26 @@ TEST_F(SliceStringTest, StringWithNulls) {
     }
 }
 
+TEST_F(SliceStringTest, StringWithNullsColumn) {
```

Comment:
Along with this add test for non-string just as back-up test case.

Reply:
done @rgsl888prabhu https://github.com/rapidsai/cudf/pull/3886/commits/ccb09bff29ccdae97498d71d716ddc6ea0390e05

Reply:
note I did not add one for table vs table view, I have not run into this issue with table slicing (I believe it might be redundant to add a table slice test?)

Reply:
I agree as it would be just the column operations bunched together.